homeassistant/components/envisalink/alarm_control_panel.py

```diff
@@ -117,6 +117,7 @@ def __init__(
         self._partition_number = partition_number
         self._code = code
         self._panic_type = panic_type
+        self._attr_unique_id = f"{partition_number}"
```

Comment:
This is not unique enough, there is nothing to distinguish the device. 

When this is ported to a config flow, it could leverage the config entry ID as a base unique ID for each device.

Reply:
Thank you for the feedback @balloob, I will see what else can be added to this.

Reply:
I will close this pull request and look into adding a config flow.